xsd/netex_framework/netex_reusableComponents/netex_equipment_support.xsd

```diff
@@ -116,7 +115,7 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:element name="TypeOfEquipmentRef" type="TypeOfEquipmentRefStructure" substitutionGroup="TypeOfEntityRef">
 		<xsd:annotation>
-			<xsd:documentation>Reference to a TYPE OF EQUIPMENT.</xsd:documentation>
+			<xsd:documentation>Reference to a TYPE OF EQUIPMENT. Some values are defined for EPIAP "mobileRamp" for mobile Ramps, "permanentRamp" for permanent installation and "wheelchairLift" for a hoist in LiftEquipment.</xsd:documentation>
```

Comment:
wheelchairLift is defined for AccessFacilitySet, not for LiftEquipment
The mentioned tokens do not refer to real EQUIPMENTS like LiftEquipment, RampEquipment - this confuses me because they should be References to a TYPE OF EQUIPMENT, though perhaps that's perfectly correct...

Reply:
I think this should be fine. This is not a direct reference to the type, but an enum in association.  Or do the others think it should be more aligned? @duexw @Aurige @skinkie 
We currently don't have an example. Do we need one?